woodwork/table_accessor.py

```diff
@@ -38,28 +41,34 @@
 dd = import_or_none('dask.dataframe')
 ks = import_or_none('databricks.koalas')
 
+if TYPE_CHECKING:
+    from woodwork.typing import AnyDataFrame, ColumnName, UseStandardTagsDict  # pragma: no cover
+
 
 class WoodworkTableAccessor:
     def __init__(self, dataframe):
         self._dataframe_weakref = weakref.ref(dataframe)
         self._schema = None
 
-    def init(self,
-             index=None,
-             time_index=None,
-             logical_types=None,
-             already_sorted=False,
-             schema=None,
-             validate=True,
-             use_standard_tags=True,
-             **kwargs):
-        """Initializes Woodwork typing information for a DataFrame.
+    def init(self, **kwargs):
+        """Initializes Woodwork typing information for a DataFrame with a partial schema.
+        Logical type priority:
+        1. Types specified in ``logical_types``
+        2. Types specified in ``partial_schema``
+        3. Types inferred by ``ww.type_system.infer_logical_type``
+
+        Other Info priority:
+        1. Parameter passed in
+        2. Value specified in ``partial_schema``
```

Comment:
![image](https://user-images.githubusercontent.com/64278226/130511814-f8f4332b-1d37-43f1-8969-bacf1a3c0f12.png)

Formatting here and above isn't coming through in the docs build

Reply:
The formatting should be fixed! Is it ok if I keep the kwargs as` **kwargs`?

Reply:
looks good! Also I'm fine keeping the kwargs as they are in `init`!